### ISSUE TITLE
Seeing Page full of texts instead of camera stream

### ISSUE THREAD

**Issue by Priya4120** [silvanmelchior/RPi_Cam_Web_Interface#676]
I am able to install and run the module. however, when I reach the webpage I see a page for of text instead of the camera stream. I checked with the raspistill that my camera is working
![Screenshot 2022-08-13 142904](https://user-images.githubusercontent.com/64700114/184506449-bb8f644a-f4e4-40ab-aa2e-5d17defd4ee2.png)
.

**Comment by roberttidey** [comment]
This is probably an issue with browser set up. For example if javascript has been disabled.

What browser are you using? Have you tied accessing from elsewhere like from a PC or phone?

For most browsers you can turn on developer options (F12) which will show what is going on.

**Comment by Priya4120** [comment]
Thanks for the response. I tried accessing the developer options using F12. I am using chrome and Java script is enabled. Here is my output of the console and issues within developer options. Any help is appreciated. 

![console](https://user-images.githubusercontent.com/64700114/184544723-95605140-e8fe-42a4-a110-d0e904f50dff.png)
![issues](https://user-images.githubusercontent.com/64700114/184544726-7ad9403a-492b-4aca-9d03-56f00da4652b.png)

**Comment by roberttidey** [comment]
Have you got chrome speechify extension?

If so try disabling this.

**Comment by Priya4120** [comment]
That's right. I removed speechify extension. Issues are cleared now but I am still seeing some console errors. Sorry, I am not much experienced on web side. 

![image](https://user-images.githubusercontent.com/64700114/184545939-4fb70c30-3791-48b9-b38b-edb86db6ddb2.png)

**Comment by roberttidey** [comment]
I'll look further into those. I don't get any console errors.

There are references to port 82 and port 291 in your errors> Have you changed port numbers?

init_pt which is referenced doesn't exist in my code.

**Comment by Elite230** [comment]
Hi, I am also getting a similar error on chrome without extensions...

**Comment by roberttidey** [comment]
I don't see an issue using latest software and using chrome as browser.

Check by turning on developer tools (F12) and looking for errors in console.
